apps/token-manager/contracts/test/mocks/TokenManagerMock.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.4.24;
+
+import "../../TokenManager.sol";
+
+
+contract TokenManagerMock is TokenManager {
+    uint64 mockTime = uint64(now);
```

Comment:
When using this contract behind a proxy, this value will not be set and `mockTime` will be 0 until `mock_setTimestamp` is used